lib/vagrant-proxmox/proxmox/connection.rb

```diff
@@ -184,12 +184,12 @@ def qemu_agent_get_vm_ip(node, vm_id)
 
         response = post "/nodes/#{node}/qemu/#{vm_id}/agent", command: "network-get-interfaces"
         response[:data][:result]
-          .select { |iface| iface[:name] != 'lo' }
-          .map { |iface| iface[:'ip-addresses'] }
-          .flatten
-          .select { |ip| ip[:'ip-address-type'] == 'ipv4' }
-          .map { |ip| ip[:'ip-address'] }
-          .first
+        .select { |iface| iface[:name] != 'lo' }
+        .map { |iface| iface[:'ip-addresses'] }
+        .flatten
+        .select { |ip| ip[:'ip-address-type'] == 'ipv4' }
+        .map { |ip| ip[:'ip-address'] }
+        .first
```

Comment:
I'd say that these should stay indented, so that we know they're part of the chain and not separate statements.